src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -51,16 +55,159 @@ graph_status_to_string(int status) {
   }
 }
 
+void
+populate_aie_core(boost::property_tree::ptree _pt,boost::property_tree::ptree& tile, int row, int col)
+{
+  boost::property_tree::ptree pt;
+  std::string aie_data;
+  std::vector<std::string> graph_status;
+  boost::property_tree::ptree tile_array;
+  boost::property_tree::ptree empty_pt;
+  try {
+    pt =  _pt.get_child("aie_core."+std::to_string(col)+"_"+std::to_string(row));
+    tile.put("column", pt.get<uint32_t>("col"));
+    tile.put("row", pt.get<uint32_t>("row"));
+    int count = 0;
+    std::string status;
+    for (auto& node: pt.get_child("core.Status", empty_pt)) {
+      status +=(status.empty()?"":", ")+node.second.data();
+    }
+    if(!status.empty())
+      tile.put("core.status", status);
+
+    for (auto& node: pt.get_child("core.PC", empty_pt))
+      tile.put("core.program_counter", node.second.data());
+    for (auto& node: pt.get_child("core.LR", empty_pt))

```

Comment:
For readability, please add a new line, before each for loop.

Reply:
fixed